src/main/java/org/openapitools/codegen/languages/MicronautCodegen.java

```diff
@@ -153,12 +162,13 @@ public String getName() {
 	}
 
 	@Override
-	public void postProcess() {}
+	public void postProcess() {
```

Comment:
why reformatting? (there are other reformats in this file)

Reply:
Same as always, conflicting code formatters;) I will fix.

Reply:
done